api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/ResumoMensalQueries.java

```diff
@@ -324,6 +324,18 @@ public static String getPatientsHavingViralLoadSuppression(
     return String.format(query, viralLoadConcept, encounterType);
   }
 
+  public static String getPatientsHavingVLSuppressionOnFichaResumo() {
+    String query =
+        "SELECT p.patient_id FROM patient p JOIN encounter e ON p.patient_id=e.patient_id JOIN obs o ON e.encounter_id=o.encounter_id "
+            + " WHERE p.voided=0 AND e.voided=0 AND o.voided=0 AND e.location_id=:location "
+            + " AND o.obs_datetime BETWEEN :startDate AND :endDate "
+            + " AND o.value_numeric IS NOT NULL "
+            + " AND o.concept_id=856 "
```

Comment:
@eliazmuchave no spec and not formated as string {856
```suggestion
            + " AND o.concept_id={856} "
```